src/components/ListItem.jsx

```diff
@@ -13,7 +13,7 @@ export function ListItem({ name, listPath, id, isChecked, datePurchased }) {
 
 	useEffect(() => {
 		const today = new Date().getTime();
-		const datePurchasedInMillis = datePurchased?.toMillis();
+		const datePurchasedInMillis = datePurchased;
```

Comment:
seems like the `?.toMillis` that got deleted here is causing the items to uncheck whenever the page refreshes, which is not the desired behavior